12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,25 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
-
+	ls [a-z]*
   2. All files whose name starts with an uppercase letter
   Answer:
-
+	ls [A-Z]*
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
-
+	ls *[aeio] [u]
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
-
+	ls *[!aeio] [!u]
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
-
+	ls /src *.c
   6. All files with extension ".c" under any folder
   Answer:
-
+	ls *.c
```

Comment:
this will list all files in current directory try ```*/*.c``` (but note that it's not recursive - i,e won't work for sub directories)